### ISSUE TITLE
Is it possible to train the head of bbox on COCO? Is it possible to add a pose estimation head?

### ISSUE THREAD

**Issue by zhouhang3** [ifzhang/FairMOT#121]

**Comment by ifzhang** [comment]
Yes, you can train it on coco by setting the id to -1 so the reid head will ignore its id. The heatmap channel needs to be set 80 because COCO has 80 classes. If you just want to use the COCO-person dataset, you do not need to change the heatmap head. Pose estimation head is also possible to add because our network can estimate heatmap well.
